actionview/test/template/resolver_shared_tests.rb

```diff
@@ -79,6 +79,17 @@ def test_can_find_with_any_variant_format_and_handler
     assert_kind_of ActionView::Template::Handlers::ERB, templates[0].handler
   end
 
+  def test_can_find_when_special_chars_in_path
+    special_chars = ((" ".."~").grep(/\W/) - ["|", "/", "\\"]).join("_")
```

Comment:
I think some of these are reserved on windows. I think it would probably be sufficient for this test to just test for ex. parens since that exposes the existing issue here and feels like the most likely use.

```
dir = "my_project (testing)"
```